requirements.txt

```diff
@@ -0,0 +1,2 @@
+shtab
+python-dotenv
```

Comment:
should these be pinned to a specific version, or it doesnt matter?

Reply:
We can pin them in an abundance of caution, but given this very simple set of dependencies, I'd be interested in trying out seeing if we can avoid dependabot PRs on this `requirements.txt` file by letting them float. I think @johnclary may be able to offer us direction on what he's found most reliable in the past in his python experience.

Reply:
okay i figured these were small potatoes, but just wondered.

Reply:
having been bitten by completely untethered package versions recently, i would say to always pin unless you have a good reason not to. major versions is usually safe:

```
python-dotenv==0.*
```

Reply:
Thanks @johnclary - I appreciate your input. I'll pin these to their major versions here in a just a few while the VZDB RR updates.

Reply:
This is done - thank you both!